shub/image/build.py

```diff
@@ -67,7 +67,7 @@ def build_cmd(target, version, skip_tests):
     click.echo("The image {} build is completed.".format(image_name))
     # Test the image content after building it
     if not skip_tests:
-        test_mod.test_cmd(target, version)
+        test_cmd.test_cmd(target, version)
```

Comment:
test_module can be anything, test_cmd is a test command, I think it's just easier to digests.